### ISSUE TITLE
Will it work on current HA versions

### ISSUE THREAD

**Issue by Miahbird** [mrk-its/homeassistant-sds011#3]
I can't seem to make it work even with the manifest.json file that was made fix in the HA version. Is this still useable for current versions? @mrk-its

**Comment by rubinho101** [comment]
yes - you just need to add the manifest.json incl. "homeassistant": "yourVersion" and "version":"0.1".

BR
